.packit.yaml

```diff
@@ -34,36 +42,20 @@ jobs:
     targets:
       - epel-9-x86_64
       - epel-9-aarch64
+      - epel-9-ppc64le
+      - epel-9-s390x
       - epel-10-x86_64
       - epel-10-aarch64
+      - epel-10-ppc64le
+      - epel-10-s390x
       - fedora-40-x86_64
       - fedora-40-aarch64
+      - fedora-40-ppc64le
+      - fedora-40-s390x
       - fedora-41-x86_64
       - fedora-41-aarch64
-    actions:
-      # bump spec so we get release starting with 2 and hence all the default branch builds will
-      # have higher NVR than all the PR builds
-      post-upstream-clone:
-        - rpmdev-bumpspec --comment='latest upstream build' ./packaging/command-line-assistant.spec
```

Comment:
@r0x0d was this removed on purpose?

Reply:
Yes, that was targeting develop branch, which does not exist anymore.